src/pyFAI/third_party/_local/transformations.py

```diff
@@ -779,7 +779,7 @@ def decompose_matrix(matrix):
     True
 
     """
-    M = numpy.array(matrix, dtype=numpy.float64, copy=True).T
+    M = numpy.asarray(matrix, dtype=numpy.float64, copy=True).T
```

Comment:
`asarray` does not have a `copy` argument with `numpy` v1, `np.array` behavior didn't change with `copy=True`:

```suggestion
    M = numpy.array(matrix, dtype=numpy.float64, copy=True).T
```